### PR TITLE
workflow house cleaning June

### PR DESCRIPTION
**Pull request by mwdchang**
### Summary
- Remove unused/unimplemented buttons: reset zoom and reset layout
- Remove send to front/back that are unimplemented
- Clean up auto-output propagation

### CODE REVIEW

File: packages/client/hmi-client/src/components/operator/tera-operator-header.vue
Comment (YohannParis) [inline packages/client/hmi-client/src/components/operator/tera-operator-header.vue:1]: Is it coming back in the future, or can we remove this feature cleanly?

File: packages/client/hmi-client/src/components/workflow/tera-workflow.vue [hunk 1]
```diff
@@ -41,6 +41,7 @@
 				</div>
 				<Menu ref="optionsMenu" :model="optionsMenuItems" :popup="true" />
 				<div class="button-group">
+					<!--
```
Comment (YohannParis) [inline packages/client/hmi-client/src/components/workflow/tera-workflow.vue:44]: Why not remove the code, like the functions below?

**Review by YohannParis — APPROVED** [review]
LGTM

**Review by Tom-Szendrey — APPROVED** [review]
Ty for the house cleaning

File: packages/client/hmi-client/src/components/operator/tera-operator-header.vue
Comment (mwdchang) [reply packages/client/hmi-client/src/components/operator/tera-operator-header.vue:1]: Neil ask them to be hidden, but yeah good point, might as well. Easy enough to bring things back.

File: packages/client/hmi-client/src/services/workflow.ts [hunk 2]
```diff
@@ -260,6 +260,12 @@ export function getPortLabel({ label, type, isOptional }: WorkflowPort) {
 	return portLabel;
 }
 
+// Checker for resource-operators (e.g. model, dataset) that automatically create an output
+// without needing to "run" the operator because we can drag them onto the canvas
+export function canPropagateResource(outputs: WorkflowOutput<any>[]) {
+	return _.isEmpty(outputs) || (outputs.length === 1 && !outputs[0].value);
+}
```
Comment (shawnyama) [inline packages/client/hmi-client/src/services/workflow.ts:267]: This pattern is used in non resource operators as well. The input is typically the first output we want to propagate. I use it in model-edit

**Review by shawnyama — APPROVED** [review]

File: packages/client/hmi-client/src/services/workflow.ts (hunk 2 above)
Comment (mwdchang) [reply packages/client/hmi-client/src/services/workflow.ts:267]: I don't see any append-output-port in `tera-model-edit-node`

Comment (shawnyama) [reply packages/client/hmi-client/src/services/workflow.ts:267]: It's in the drilldown

Comment (mwdchang) [reply packages/client/hmi-client/src/services/workflow.ts:267]: Gotcha, I would say that is kind of a special case then - that you would propagate when opening the drilldown. Will keep a note on it and revisit.

**Pull request merged**